packages/frontend-shared/src/locales/en-US.json

```diff
@@ -665,6 +666,10 @@
       "gitRepositoryNotDetected": "Git repository not detected",
       "ensureGitSetupCorrectly": "Cypress uses Git to associate runs with your local state. Please ensure that version control is set up correctly."
     },
+    "pending": {
+      "title": "Testing",
+      "pendingCount": "{completed} of {total} specs completed | {completed} of {total} spec completed | {completed} of {total} specs completed"
```

Comment:
What is the third option here for?

Reply:
0, 1, or multiple. I suppose the 0 option doesn't make sense given the data possibilities so I could consolidate into a 1 and multiple, but did all three for the sake of completeness